src/Uno.UI.Wasm/WasmScripts/Uno.UI.d.ts

```diff
@@ -761,6 +761,11 @@ declare namespace Windows.UI.Core {
         private clearStack;
     }
 }
+interface Navigator {
```

Comment:
Seems like this is not within the scope of this commit/issue?

Reply:
This is going to be committed by someone else eventually, because it's a generated file. It's fine to leave it there :)